README.md

```diff
@@ -287,6 +287,46 @@ onmessage = (event) => {
 };
 ```
 
+## Node.js
+
+You can run highlight.js on node to prepare content before sending it to the browser.
+
+
+```js
+// require the highlight.js library including all languages
+const hljs = require('./highlight');
+const highlightedCode = hljs.highlightAuto('<span>Hello World!</span>').value
+```
+
+```js
+// require the highlight.js library without languages
+const hljs = require("highlight.js/lib/highlight.js");
+// seperatly require languages
+hljs.registerLanguage('html', require('highlight.js/lib/languages/html'));
+hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'));
+// highlight with providing the language
+const highlightedCode = hljs.highlight('html', '<span>Hello World!</span>').value
+```
+
+### highlight() return value
+```js
```

Comment:
Should there be a line break here or does it really not matteR?

Reply:
You mean above the code block, correct? I added one.